### PR TITLE
Update concurrency

### PR DESCRIPTION
**Pull request by flamingbear**
Still received status 429 from lambda just running the dashboard.

{
    "Reason": "ReservedFunctionConcurrentInvocationLimitExceeded",
    "Type": "User",
    "message": "Rate Exceeded."
}

**Summary:** Summary of changes

Addresses [CUMULUS-2695](https://bugs.earthdata.nasa.gov/browse/CUMULUS-2695)

## PR Checklist

- [X] Update CHANGELOG
- [ ] Unit tests
- [X] Ad-hoc testing - Deploy changes and test manually
- [ ] Integration tests

### CODE REVIEW

File: example/cumulus-tf/variables.tf [hunk 1]
```diff
@@ -166,7 +166,7 @@ variable "api_gateway_stage" {
 
 variable "api_reserved_concurrency" {
   type = number
-  default = 8
+  default = 15
```
Comment (markdboyd) [inline example/cumulus-tf/variables.tf:169]: If this value is configurable, then why do we need to update our default? DAACs can update this value without us changing it in core.

I suppose the argument is that our default value should be sufficient to serve the dashboard?

Comment (flamingbear) [reply example/cumulus-tf/variables.tf:169]: Yeah actually they should, but most won't configure it, and I got hit with just 8.

Comment (Jkovarik) [reply example/cumulus-tf/variables.tf:169]: In that case, should we be updating the default in `tf-modules/cumulus/variables.tf` and possibly removing this?

Comment (markdboyd) [reply example/cumulus-tf/variables.tf:169]: I would think that we still might want it configurable for example deployments, but yeah we definitely want to update the default in `tf-modules/cumulus/variables.tf`

Comment (flamingbear) [reply example/cumulus-tf/variables.tf:169]: Uh yup, didn't remember/know about that. 99bcdc1c74

Comment (flamingbear) [reply example/cumulus-tf/variables.tf:169]: whoops, wrong branch....

Comment (flamingbear) [reply example/cumulus-tf/variables.tf:169]: f82cb293a6

File: CHANGELOG.md [hunk 2]
```diff
@@ -26,6 +26,11 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
   issue in previous version.
 - **CUMULUS-2583**
   - `QueueGranules` task now updates granule status to `queued` once it is added to the queue.
+- **CUMULUS-2695**
+  - Updates the example/cumulus-tf deployment to change
```
Comment (markdboyd) [inline CHANGELOG.md:30]: Should be updated to reflect that the default int he user facing `tf-modules/cumulus` module was changed

Comment (flamingbear) [reply CHANGELOG.md:30]: e5b03ca35f

**Review by markdboyd — APPROVED** [review]

**Comment by nemreid** [comment]
Only one note here: Note that reserved concurrency *reserves* concurrency for that function and that function alone, and is subtracted from the account limit. Going much higher than this may introduce problems with our sandbox lambda limits as 15 * 2 (deployments per dev) * devs may become a very high number.

**Comment by flamingbear** [comment]
@nemreid yes, that's a good point, maybe we should leave the cumulus value at 15 so that DAACs don't see the limit, and change our default to something smaller.  I don't think the dash works great with less than 8 though.  but that said, the dash testers/devs can also override in our stacks that we're working with.

cumulus/variables.tf => 15
example/cumulus-tf => 8 (4?)
Dev users => whatever they need

**Comment by nemreid** [comment]
I am content with any and all of those suggestions as long as we are aware of the problem. 👍

**Pull request merged**